template-parts/content.php

```diff
@@ -58,7 +58,7 @@
 		?>
 	</div><!-- .entry-content -->
 
-	<footer class="entry-footer">
+	<footer class="entry-footer container">
```

Comment:
```suggestion
	<footer class="entry-footer">
```

Container is already applied